llvm/test/CodeGen/AMDGPU/amdgpu-oclc-reflect.ll

```diff
@@ -0,0 +1,16 @@
+; RUN: opt -S -p amdgpu-oclc-reflect %s | FileCheck %s -check-prefixes=CHECK,CHECK-SAFE-ATOMICS
```

Comment:
Could this be updated to use `update_test_checks.py`? Might be better than manually checking for what we want?

Reply:
Aha TIL. Thanks have updated

Reply:
Thanks!